user-operator/src/main/java/io/strimzi/operator/user/operator/SimpleAclOperator.java

```diff
@@ -38,7 +37,7 @@
  * SimlpeAclOperator is responsible for managing the authorization rules in Apache Kafka / Apache Zookeeper.
  */
 public class SimpleAclOperator {
-    private static final Logger log = LogManager.getLogger(SimpleAclOperator.class.getName());
+    private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(SimpleAclOperator.class.getName());
```

Comment:
Same as above ... this looks like it doesn't need `ReconciliationLogger` right now. But I think it should be really using it since it is part of a reconciliation.